lib/src/charts/painters/cartesian/bar_painter.dart

```diff
@@ -4,35 +4,38 @@ import 'package:chart_it/src/charts/painters/cartesian/cartesian_chart_painter.d
 import 'package:chart_it/src/charts/painters/cartesian/cartesian_painter.dart';
 import 'package:chart_it/src/charts/painters/text/chart_text_painter.dart';
 import 'package:chart_it/src/extensions/paint_objects.dart';
+import 'package:chart_it/src/extensions/primitives.dart';
 import 'package:flutter/material.dart';
 
 class BarPainter implements CartesianPainter {
+  late BarSeriesConfig _config;
   late BarSeries _data;
+
   late double _vRatio;
   late double _unitWidth;
   final bool useGraphUnits;
-  final int maxBarsInGroup;
+
+  // final int maxBarsInGroup;
 
   /// get the instance of our painter
   static BarPainter? _instance;
```

Comment:
We don't need to maintain this static instance anymore. Putting it in the map will take care of maintaining a single instance.
Let's remove this `_instance`